lawnchair/src/app/lawnchair/preferences2/PreferenceManager2.kt

```diff
@@ -192,6 +192,12 @@ class PreferenceManager2(private val context: Context) : PreferenceManager {
         onSet = { reloadHelper.recreate() }
     )
 
+    val showHiddenAppsInSearchBar = preference(
```

Comment:
`showHiddenAppsInSearch` would be more appropriate, as the ui is called "search" and not "search bar". 
This also applies to other variables too.